scripts/west_commands/runners/openocd.py

```diff
@@ -118,14 +118,30 @@ def do_flash(self, **kwargs):
             raise ValueError('Cannot flash; verify command is missing')
 
         self.logger.info('Flashing file: {}'.format(self.hex_name))
+
+        pre_init_cmd = []
+        pre_load_cmd = []
+        post_verify_cmd = []
+        for i in self.pre_init:
```

Comment:
I'm not 100% sure what you actually try to do but if you are trying to insert an element in the start of the list if it's not empty I think it's cleaner to write:

```python
if self.pre_init:
    self.pre_init.insert(0, "-c")
```

since I assume this is not performance critical code. Instead of creating new temp variables

Reply:
I am adding "-c" for every list element, so something like this:

`["a", "b", "c"]`

will turn into 

`["-c", "a", "-c", "b", "-c", "c"]`

I am sure there different ways to do this in python, the above is the most straightforward.

Reply:
If you don't mind having it combined with the argument
```python
['-c ' + item for item in list]
```

Reply:
that was my original change, however, the quoting was wrong in the resulting command being issued.

Reply:
Sorry I edited my comment so I'm unsure to which of these you are referring I'm guessing
```
self.pre_init = ['-c ' + item for item in self.pre_init]
```

Reply:
> Sorry I edited my comment so I'm unsure to which of these you are referring I'm guessing

yes